woff1/tools/validator/validator.py

```diff
@@ -1801,8 +1801,8 @@ def sumDataULongs(data):
     return value
 
 def calcChecksum(tag, data):
-    if tag == "head":
-        data = data[:8] + "\0\0\0\0" + data[12:]
+    if tag.decode() == "head":
```

Comment:
Binary data needs to be decoded in order to be able to perform string comparison in Python 3.